.vscodeignore

```diff
@@ -1,8 +1,8 @@
 !ReactTypings/**
 !js-patched/**
 .vscode/**/*
+.vscode-test/**/*
```

Comment:
Yes; when running the tests via "npm test", VSCode's default test launcher uses this folder.